src/frontend/screens/ExportObservations.tsx

```diff
@@ -0,0 +1,210 @@
+              title={formatMessage(m.allObservations)}
+              description={formatMessage(m.allObservationsDescription)}
+              isSelected={typeToExport === 'Observation'}
+              showError={showError && !typeToExport}
+              onPress={() => {
+                setTypeToExport('Observation');
+                if (showError) {
+                  setShowError(false);
+                }
+              }}
+            />
+            <ExportOptionCard
+              title={formatMessage(m.allObservationsAndMedia)}
+              description={formatMessage(m.allObservationsAndMediaDescription)}
+              isSelected={typeToExport === 'ObservationsWithMedia'}
+              showError={showError && !typeToExport}
+              onPress={() => {
+                setTypeToExport('ObservationsWithMedia');
+                if (showError) {
+                  setShowError(false);
+                }
+              }}
+            />
+          </>
+        )}
+        {tracks.length > 0 && (
+          <ExportOptionCard
+            title={formatMessage(m.tracks)}
+            description={formatMessage(m.tracksDescription)}
+            isSelected={typeToExport === 'Tracks'}
+            showError={showError && !typeToExport}
+            onPress={() => {
+              setTypeToExport('Tracks');
+              if (showError) {
+                setShowError(false);
+              }
+            }}
+          />
+        )}
+      </View>
+      {showError && (
+        <HeaderText
+          variant="header5"
+          style={{
+            color: WARNING_RED,
+            marginTop: 20,
+            textAlign: 'center',
+          }}>
+          {formatMessage(m.errorMessage)}
+        </HeaderText>
+      )}
+      {!exportAndShare.isPending ? (
+        <>
+          <PrimaryButton
+            fullSize={true}
+            onPress={handlePressDownload}
+            style={{marginTop: 20, alignSelf: 'center'}}
+            text={formatMessage(m.download)}
+          />
+          <SecondaryButton
+            fullSize={true}
+            onPress={() => navigation.goBack()}
+            style={{marginTop: 20, alignSelf: 'center'}}
+            text={formatMessage(m.close)}
+          />
+        </>
+      ) : (
+        <UIActivityIndicator style={{paddingTop: 60, paddingBottom: 40}} />
+      )}
+    </BottomSheetWrapper>
+  );
+};
+
+type ExportOptionCardProps = {
+  title: string;
+  description: string;
+  isSelected: boolean;
+  showError: boolean;
+  onPress: () => void;
+};
+
+export const ExportOptionCard = ({
+  title,
+  description,
+  isSelected,
+  showError,
+  onPress,
+}: ExportOptionCardProps) => {
+  return (
+    <TouchableOpacity style={styles.cardButton} onPress={onPress}>
+      <MaterialIcon
+        name={isSelected ? 'radio-button-checked' : 'radio-button-unchecked'}
+        color={showError ? WARNING_RED : DARK_GREY}
+        size={30}
+        style={{marginRight: 10}}
+      />
+      <View>
```

Comment:
If you want to address the padding issue mentioned [here](https://github.com/digidem/comapeo-mobile/pull/1192/files#r2103326824)

```suggestion
      <View style={{flex: 1}}>
```